src/Ether.Core/Types/Handlers/Commands/GenerateAggregatedWorkitemsETAReportHandler.cs

```diff
@@ -161,5 +163,51 @@ private async Task<List<TeamMemberViewModel>> GetAllTeamMembers(IDataSource data
 
             return allMembers;
         }
+
+        private int WorkdaysAmount(DateTime start, DateTime end)
```

Comment:
There is already a function that calculates business days here https://github.com/BerserkerDotNet/Ether/blob/bc6bd7ace31d3087ebfbc5d18a676077a7fc7101/src/Ether.Vsts/Types/VstsDataSource.cs#L417

Refactor to re-use it.